docs/pages/versions/v37.0.0/sdk/battery.md

```diff
@@ -30,7 +30,7 @@ export default class App extends React.Component {
     batteryLevel: null,
   };
 
-  componentDidMount() {
+  async componentDidMount() {
```

Comment:
Need to apply the change to the unversioned file as well. Otherwise it looks good!

Reply:
done